packages/common/vitest.config.ts

```diff
@@ -1,12 +1,14 @@
-import { defineProject, mergeConfig } from "vitest/config";
-import { sharedConfig } from "../../.vitest/vitest.shared";
+import { defineProject } from "vitest/config";
 
-export default mergeConfig(
-  // @ts-ignore this does work
-  sharedConfig,
-  defineProject({
-    test: {
-      name: "alchemy/common",
-    },
-  }),
-);
+export default defineProject({
```

Comment:
not sure if this is the correct way to go about it. the existing config was spinning up anvil and failing, but we don't have any integration tests at this level. everything should be unit testable.